thirdparty/php/zimbra-php/debian/changelog

```diff
@@ -1,3 +1,9 @@
+zimbra-php (VERSION-1zimbra8.7b3ZAPPEND) unstable; urgency=medium
+
+  * Upgraded PHP to 7.4.27 (ZBUG-2498)
+
+   -- Zimbra Packaging Services <packaging-devel@zimbra.com>  Mon, 14 Feb 2022 21:56:52 +0000
```

Comment:
nit: fix indent. Use a single white space as same as other "-- xxx" descriptions.
```suggestion
 -- Zimbra Packaging Services <packaging-devel@zimbra.com>  Mon, 14 Feb 2022 21:56:52 +0000
```

Reply:
Thanks, I have fixed this